iocore/net/UnixNetVConnection.cc

```diff
@@ -1514,9 +1514,10 @@ UnixNetVConnection::set_inactivity_timeout(ink_hrtime timeout_in)
 #else
   if (timeout_in == 0) {
```

Comment:
Does not need it, since #771 was reverted.
The code changes by this PR is a part of #771 ,  It was lost during code merge from our internal branch.

Because our internal branch is saved in a separate network, I copy the code from "another screen" to "my local screen".